expression_prediction_evaluation.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<blockquote>We want to evaluate how the prediction performs in relation to both groups: control condition input into the model and the target perturbed condition.</blockquote><p>I find this sentence a bit hard to understand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>